docs/release-notes.txt

```diff
@@ -8,6 +8,44 @@
  Version 0.13
 **************
 
+Version 0.13.13
+===============
+
+**Release Date:** January 21, 2021
+
+**New Features**
+
+-  Update experiment details pages to include a learning curve
+   visualization. This will enable a comparison of hyperparameter
+   performance among many different trials within an experiment.
+
+-  Support Elasticsearch as an alternative backend for logging. Read
+   more about :ref:`elasticsearch-logging-backend` to see if it's
+   appropriate for your Determined deployment.
+
+**Improvements**
+
+-  WebUI: Enable filtering of trial logs by agent, container, rank, log
+   level, and timestamp.
+
+-  WebUI: Improve section contrast on all pages.
```

Comment:
Not sure if this is release note worthy - will any of our users like to know this?

Reply:
I thought the same, but I remember getting some complaints that everything is "too white" so this might be good news to some folks.

Reply:
Yeah, people (internal and external) have complained that it's "too white", so it'd be nice to know that we truly care about not hurting your eyes :)